labgrid/driver/modbusdriver.py

```diff
@@ -15,13 +16,18 @@ class ModbusCoilDriver(Driver, DigitalOutputProtocol):
     def __attrs_post_init__(self):
         super().__attrs_post_init__()
         self._module = import_module('pyModbusTCP.client')
-        port = "502"
-        host = self.coil.host
-        if ':' in host:
-            host, port = host.split(':', 1)
+        self.client = None
+
+    def on_activate(self):
+        import onewire
```

Comment:
Why is this needed in the ModbusCoilDriver?

Reply:
This was a copy/paste error.